Gordon360/Models/ViewModels/Class.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Gordon360.Models.ViewModels
+{
+    public class OfficeLocationPatchViewModel
+    {
+        public string BuildingDescription { get; set; }
+        public string RoomNumber { get; set; }
+    }
+}
```

Comment:
```suggestion
﻿namespace Gordon360.Models.ViewModels
{
    public class OfficeLocationPatchViewModel
    {
        public string BuildingDescription { get; set; }
        public string RoomNumber { get; set; }
    }
}
```

Good viewmodel, but you need to rename the file from `Class.cs` to `OfficeLocationPatchViewModel`

Reply:
This was marked as resolved, but there has not been a push to show changes?

Reply:
my bad, I remembered renaming it in edit-office-location branch and thought I also changed this one. just fixed it!